tests/dir-misc/cursor_restore.lua

```diff
@@ -0,0 +1,61 @@
+test1 = function()
```

Comment:
You certainly shouldn't need this large file for test purposes. Even to test centering.

Reply:
Fixed.